research/src/sources/fabric.json5

```diff
@@ -490,6 +490,24 @@
     {
       "name": "Themes",
       "url": "https://developer.microsoft.com/en-us/fabric#/controls/web/themes"
-    }
+    },
+    {
+      name: 'Menu List',
+      url: 'https://semantic-ui.com/collections/menu.html',
```

Comment:
Do we not have a Fluent UI link for this yet? As an additional note, there are now multiple web implementations of Fluent UI including a web component instance. How do we determine what we pull research from? Considering that the core foundation element of the Fluent UI web component is in FAST (https://fast.design) one way to ensure both are captured in the research would be to include FAST as part of this PR rather than having multiple implementations of Fluent UI shown.

Reply:
That's an unfortunate typo. I've fixed the link.

You make a fair point about FAST as an upcoming flavour of fluent. I've added the control and concepts of the FAST menu component to this PR